Description: Add a setup.py
Author: Steev Klimaszewski <steev@kali.org>
Origin: Kali Linux, https://gitlab.com/kalilinux/packages/sherlock/-/raw/fbc28d302bd78203f7cf3449c99bca700ecd889a/debian/patches/Add-a-setup.py.patch
Forwarded: not-needed
Last-Update: 2023-02-16
Index: sherlock/setup.py
===================================================================
--- /dev/null
+++ sherlock/setup.py
@@ -0,0 +1,30 @@
+import setuptools
+import re
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+
+with open('sherlock/sherlock.py', 'rt', encoding='utf8') as f:
+    version = re.findall('__version__ = "(.*)"', f.read())[0]
+
+setuptools.setup(
+    name="sherlock",
+    version=version,
+    author="Sherlock Project",
+    description="Hunt down social media accounts by username across social networks",
+    url="https://github.com/sherlock-project/sherlock",
+    packages=setuptools.find_packages(exclude=['sherlock.tests']),
+    package_data={"": ["resources/data.json"]},
+    include_package_data=True,
+    python_requires=">=3.0",
+    install_requires=requirements,
+    classifiers=[
+        "Operating System :: OS Independent",
+        "Programming Language :: Python ::3",
+        "Environment :: Console",
+        "License :: OSI Approved :: MIT License",
+    ],
+    entry_points={
+        "console_scripts": ["sherlock=sherlock.sherlock:main"],
+    },
+)
