Description: Fixed module import path
Author: Josenilson Ferreira da Silva <nilsonfsilva@hotmai.com>
Forwarded: https://github.com/sherlock-project/sherlock/issues/1939
Last-Update: 2023-11-09
Index: sherlock/sherlock/tests/test_multiple_usernames.py
===================================================================
--- sherlock.orig/sherlock/tests/test_multiple_usernames.py
+++ sherlock/sherlock/tests/test_multiple_usernames.py
@@ -1,8 +1,9 @@
-import importlib
+import importlib.util
 import unittest
+import os
 import sys
-sys.path.append('../')
-import sherlock as sh
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../')))
+from sherlock import sherlock as sh
 
 checksymbols = []
 checksymbols = ["_", "-", "."]
@@ -12,7 +13,7 @@ checksymbols = ["_", "-", "."]
         This test ensures that the function multiple_usernames works properly. More specific,
         different scenarios are tested and only usernames that contain this specific sequence: {?} 
         should return positive.
-      
+
         Keyword Arguments:
         self                   -- This object.
 
@@ -26,4 +27,4 @@ class TestMultipleUsernames(unittest.Tes
             if(sh.check_for_parameter(name)):
                 self.assertAlmostEqual(sh.multiple_usernames(name), ["test_test" , "test-test" , "test.test"])
             else:
-                self.assertAlmostEqual(name, name)
\ No newline at end of file
+                self.assertAlmostEqual(name, name)
